orttraining/orttraining/python/training/ortmodule/_execution_agent.py

```diff
@@ -2,27 +2,40 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
-
 import onnxruntime
 from onnxruntime.capi import _pybind_state as C
-from onnxruntime.capi.onnxruntime_inference_collection import IOBinding, OrtValue
+from onnxruntime.capi.onnxruntime_inference_collection import IOBinding, get_ort_device_type
 from onnxruntime.capi._pybind_state import TrainingAgent as C_TrainingAgent
 
+from . import _utils
+from ._graph_execution_manager import RunStateInfo
+
+import torch
+from torch.utils.dlpack import to_dlpack
 
 class ExecutionAgentOutput(object):
     def __init__(self, ortvalues, run_id=None):
         self.ortvalues = ortvalues
         self.run_id = run_id
 
+class ExecutionAgent(object):
+    def __init__(self, onnx_model, device):
+        """
+        :param onnx_model: ONNX ModelProto
```

Comment:
ORTModule uses google code style for python code

Reply:
Is the new version better?

Reply:
@thiagocrepaldi shall I also remove the `:param` annotation?

Reply:
done